array/models/actions/GetContactByName.model.bxb

```diff
@@ -0,0 +1,11 @@
+action (GetContactByName) {
+  description (fetch contact structure by contact name)
```

Comment:
maybe just simply say "Search contacts by name"? Using the word "fetch" on a "search" action can be confusing.

Reply:
Done.